### DISCUSSION TITLE
Has QUIC_PARAM_CONN_SEND_BUFFERING been deprecated?

### DISCUSSION THREAD

**Discussion by murphypei** [microsoft/msquic#3237] [Q&A] [answered]
Hi, I am new to msquic, in the stream doc, I have got a way to enable no buffer for sending.

> To enable this mode, the app must call [SetParam](https://github.com/microsoft/msquic/blob/main/docs/api/SetParam.md) on the connection with the QUIC_PARAM_CONN_SEND_BUFFERING parameter set to FALSE.

But there is no result for searching this parameter on master branch code, so how can I use it?

**Comment by murphypei** [comment, marked as answer]
Settings can do it, but docs need update.

**Reply by nibanks** [reply]
Yeah, you're right. Feel free to open a doc bug, or better yet, provide a PR to fix!

**Reply by murphypei** [reply]
thank you, I will try my best.
